docs/step-by-step/step-01.md

```diff
@@ -0,0 +1,139 @@
+    applications. We designed the dashboard with anomaly detection and quick analysis in mind. We'll return to
+    dashboard-related topics in both [step 7](step-07.md) and [step 8](step-08.md).
+-   **Netdata Cloud** is our SaaS toolkit that helps Netdata users monitor the health and performance of entire
+    infrastructures, whether they are two or two thousand (or more!) systems. We'll cover Netdata Cloud in [step
+    3](step-03.md).
+-   **No configuration necessary**. Without any configuration, you'll get thousands of real-time metrics and hundreds of
+    alarms designed by our community of sysadmin experts. But you _can_ configure Netdata in a lot of ways, some of
+    which we'll cover in [step 4](step-04.md).
+-   **Distributed, per-system installation**. Instead of centralizing metrics in one location, you install Netdata on
+    _every_ system, and each system is responsible for its metrics. Having distributed agents reduces cost and lets
+    Netdata run on devices with little available resources, such as IoT and edge devices, without affecting their core
+    purpose.
+-   **Sophisticated health monitoring** to ensure you always know when an anomaly hits. In [step 5](step-05.md), we dive
+    into how you can tune alarms, write your own alarm, and enable two types of notifications.
+-   **High-speed, low-resource collectors** that allow you to collect thousands of metrics every second while using only
+    a fraction of your system's CPU resources and a few MiB of RAM.
+-   **Long-term metrics storage**. With our new database engine, you can store days, weeks, or months of per-second
+    historical metrics. Or you can archive metrics to another database, like MongoDB or Prometheus. We'll cover all
+    these options in [step 9](step-09.md).
+
+## Why you should use Netdata
+
+Because you care about the health and performance of your systems and applications, and all of the awesome features we
+just mentioned. And it's free!
+
+All these may be valid reasons, but let's step back and talk about Netdata's _principles_ for health monitoring and
+performance troubleshooting. We have a lot of [complementary
+systems](#how-netdata-has-complementary-systems-not-competitors), and we think there's a good reason why Netdata should
+always be your first choice when troubleshooting an anomaly.
+
+We built Netdata on four principles.
+
+### Per-second data collection
+
+Our first principle is per-second data collection for all metrics.
+
+That matters because you can't monitor a 2-second service-level agreement (SLA) with 10-second metrics. You can't detect
+quick anomalies if your metrics don't show them.
+
+How do we solve this? By decentralizing monitoring. Each node collects metrics, triggers alarms, and builds dashboards
+locally. That means you can scale Netdata to any size of infrastructure and keep the power of per-second metrics.
```

Comment:
I would add to this that we try to focus on making the collection process as efficient as possible. The distributed nature of netdata is a major factor in making per-second resolution possible for large networks, but it's really the honestly amazing efficiency of data collection in the core that makes it possible on a per-system basis.

Reply:
Great point. Added some new text on this.